src/ingestion/sources/postgresql.py

```diff
@@ -21,11 +21,16 @@ def configuration_schema(cls):
             },
             "secret": [ "password" ],
         }
+    
+    def _connection_string_prefix(self):
+        return self.type
```

Comment:
If you want to keep this here and override it in redshift that's fine.

I would hardcode the prefix here though. Just because the type identifier we use happens to be the same now, doesn't mean that it won't change later and create sadness.

Reply:
hardcoded prefix to `postgresql`